src/django/api/models/boundary.py

```diff
@@ -83,3 +83,22 @@ def previous_submission(self):
             return recent_submissions[1]
 
         return None
+
+    @cached_property
+    def official_name(self):
+        """
+        The official name for the boundary.
+
+        In the format: {PWSID}_{UtilityName}_{SubmittedDate}
+
+        If the boundary is not submitted yet,
+        then it is in the format: {PWSID}_{UtiltyName}_UNSUBMITTED
+        """
```

Comment:
Normally I'm not huge fan of comments which don't say much more than what the function already says clearly, but in this case I'm guessing it's for IDE hinting and not explaining the code below.

Reply:
This was done out of habit, to outline for myself what the function would do, before I wrote out the code. Didn't know how complex the code would've been, so started with this. But since this is already outdated (mentions `UtilityName` whereas we're using `utility.compact_name`), will remove.

Reply:
[2d9e571](https://github.com/azavea/iow-boundary-tool/pull/202/commits/2d9e571dccd9624fda44e6d4f83174342c3dea92)